jdbc/src/test/java/com/interface21/jdbc/core/JdbcTemplateTest.java

```diff
@@ -35,6 +37,7 @@ void setup() throws SQLException {
 
         given(dataSource.getConnection()).willReturn(connection);
         given(connection.prepareStatement(any())).willReturn(preparedStatement);
+        given(connection.prepareStatement(any(), anyInt())).willReturn(preparedStatement);
```

Comment:
```suggestion
        given(connection.prepareStatement(any(), anyInt())).willReturn(preparedStatement);
```

JdbcTemplate을 리팩토링하니 윗줄의 모킹이 필요없어졌네요